### DISCUSSION TITLE
How we can make space to avoid eat space pie chart?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1484] [Q&A]
Hi @vincentbernat / expert,

May i know how we can make space so the word not eat space the pie chart (as per snapshot below. For example if we have many exporter. Thanks

![{74952E89-5203-4F5D-9302-7539C6B74F54}](https://github.com/user-attachments/assets/fec2e197-36ae-48be-b2c1-37e7dd610cd2)

**Comment by vincentbernat** [comment]
Not possible.

**Reply by kronicklez** [reply]
Hi @vincentbernat,

Noted. Means the number of exporter display on pie chart cannot more than 3.

Thanks
